WordPress/Classes/ViewRelated/Blog/Blog List/BlogListDataSource.swift

```diff
@@ -82,6 +82,10 @@ class BlogListDataSource: NSObject {
 
     @objc let recentSitesMinCount = 11
```

Comment:
Honestly I'm not sure what to add here - the thing is a lot of the code around my changes could be better documented / improved / etc but the balance to avoid scope creep is a hard one.

If there's anything specific you'd like me to write, can you expand a bit?

Reply:
(The thing is that even by looking at the code, the meaning of that variable is not immediate to me...)

Reply:
that's ok, no big deal since it was already there 😄